packages/fast-layouts-react/src/utilities/breakpoint-tracker.spec.ts

```diff
@@ -0,0 +1,68 @@
+import breakpointTracker, { Breakpoints, BreakpointTrackerCallback, defaultBreakpoints, identifyBreakpoint } from "./breakpoint-tracker";
+
+describe("identifyBreakpoint", (): void => {
+    test("should return the correct breakpoint values", (): void => {
+        expect(identifyBreakpoint(0, defaultBreakpoints)).toBe(0);
+        expect(identifyBreakpoint(539, defaultBreakpoints)).toBe(0);
+        expect(identifyBreakpoint(540, defaultBreakpoints)).toBe(1);
+        expect(identifyBreakpoint(767, defaultBreakpoints)).toBe(1);
+        expect(identifyBreakpoint(768, defaultBreakpoints)).toBe(2);
+        expect(identifyBreakpoint(1083, defaultBreakpoints)).toBe(2);
+        expect(identifyBreakpoint(1084, defaultBreakpoints)).toBe(3);
+        expect(identifyBreakpoint(1399, defaultBreakpoints)).toBe(3);
+        expect(identifyBreakpoint(1400, defaultBreakpoints)).toBe(4);
+        expect(identifyBreakpoint(1778, defaultBreakpoints)).toBe(4);
+        expect(identifyBreakpoint(1779, defaultBreakpoints)).toBe(5);
+    });
+});
+
+/* tslint:disable:no-string-literal */
+describe("breakpointTracker", (): void => {
+    let subscriber: any;
+
+    beforeEach(() => {
+        subscriber = {
+            onBreakpointChanged: (notification: BreakpointTrackerCallback): void => {
+                return;
+            }
+        };
+    });
+
```

Comment:
the tests here don't really match the description - I'm not sure you really need to test for your description though, you can probably just remove these.